addons/sourcemod/scripting/vip/Debug.sp

```diff
@@ -1,111 +1,137 @@
 {
-    static char buffer[512];
-    Format(buffer, sizeof(buffer), "SQL_Response: %s", sMsg);
-    DebugMsg(buffer, ...);
+    static char szBuffer[512];
+    Format(szBuffer, sizeof(szBuffer), "SQL_Response: %s", szMsg);
+    VFormat(szBuffer, sizeof(szBuffer), szMsg, 2);
+    LogToFile(g_szDebugLogFile, szBuffer);
 }
 #else
-public void DBG_SQL_Response(const char[] sMsg, any ...) {}
+public void DBG_SQL_Response(const char[] szMsg, any ...) {}
 #endif
 
 #if defined LOG_API
-public void DBG_API(const char[] sMsg, any ...)
+public void DBG_API(const char[] szMsg, any ...)
 {
-    static char buffer[512];
-    Format(buffer, sizeof(buffer), "API: %s", sMsg);
-    DebugMsg(buffer, ...);
+    static char szBuffer[512];
+    Format(szBuffer, sizeof(szBuffer), "API: %s", szMsg);
+    VFormat(szBuffer, sizeof(szBuffer), szMsg, 2);
+    LogToFile(g_szDebugLogFile, szBuffer);
 }
 #else
-public void DBG_API(const char[] sMsg, any ...) {}
+public void DBG_API(const char[] szMsg, any ...) {}
 #endif
 
 #if defined LOG_DB
-public void DBG_Database(const char[] sMsg, any ...)
+public void DBG_Database(const char[] szMsg, any ...)
 {
-    static char buffer[512];
-    Format(buffer, sizeof(buffer), "Database: %s", sMsg);
-    DebugMsg(buffer, ...);
+    static char szBuffer[512];
+    Format(szBuffer, sizeof(szBuffer), "Database: %s", szMsg);
+    VFormat(szBuffer, sizeof(szBuffer), szMsg, 2);
+    LogToFile(g_szDebugLogFile, szBuffer);
 }
 #else
-public void DBG_Database(const char[] sMsg, any ...) {}
+public void DBG_Database(const char[] szMsg, any ...) {}
 #endif
 
-#else  // Si DEBUG_MODE no está definido, se generan funciones vacías
-public void DebugMessage(const char[] sMsg, any ...) {}
-public void DBG_Download(const char[] sMsg, any ...) {}
-public void DBG_Config(const char[] sMsg, any ...) {}
-public void DBG_Clients(const char[] sMsg, any ...) {}
-public void DBG_SQL_Query(const char[] sMsg, any ...) {}
-public void DBG_SQL_Response(const char[] sMsg, any ...) {}
-public void DBG_API(const char[] sMsg, any ...) {}
-public void DBG_Database(const char[] sMsg, any ...) {}
+char sValueType[][] = {
+	"VIP_NULL",
+	"INT",
+	"FLOAT",
+	"BOOL",
+	"STRING"
+};
+
+char sFeatureType[][] = {
+	"TOGGLABLE",
+	"SELECTABLE",
+	"HIDE"
+};
+
+char sToggleState[][] = {
+	"DISABLED",
+	"ENABLED",
+	"NO_ACCESS"
+};
+
+#else
+public void DebugMessage(const char[] szMsg, any ...) {}
+public void DBG_Download(const char[] szMsg, any ...) {}
+public void DBG_Config(const char[] szMsg, any ...) {}
+public void DBG_Clients(const char[] szMsg, any ...) {}
+public void DBG_SQL_Query(const char[] szMsg, any ...) {}
+public void DBG_SQL_Response(const char[] szMsg, any ...) {}
+public void DBG_API(const char[] szMsg, any ...) {}
+public void DBG_Database(const char[] szMsg, any ...) {}
+char sValueType[VIP_ValueType][] = { "", "", "", "", "" };
+char sFeatureType[VIP_FeatureType][] = { "", "", "" };
+char sToggleState[VIP_ToggleState][] = { "", "", "" };
```

Comment:
For string variables, use the prefix `sz`, like `szSomething`.

Why? 
Because [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation)
s -> String
sz -> Zero terminated String